pkg/build/golang/docker.go

```diff
@@ -223,11 +223,15 @@ func (b *DockerGoBuilder) Build(ctx context.Context, in *api.BuildInput, output
 		BuildOpts: &opts,
 	}
 
+	buildStart := time.Now()
+
 	err = docker.BuildImage(ctx, cli, &imageOpts)
 	if err != nil {
 		return nil, err
 	}
 
+	log.Debugw("build completed", "took", time.Since(buildStart))
```

Comment:
I think these should be `Infow`. They're not overwhelming or repetitive, and very valuable.

Reply:
Changed in https://github.com/ipfs/testground/pull/714